src/resolvers/Tasks.js

```diff
@@ -60,12 +60,21 @@ export default {
         filter,
       });
     },
-    taskGroup(parent, args, { loaders }) {
+    async taskGroup(parent, args, { loaders }) {
```

Comment:
That's an interesting name for a function that returns a decision task, and not a task group?

Reply:
You're right. We could have this renamed to `decisionTask`.